src/udev/udev-event.c

```diff
@@ -247,29 +247,29 @@ static ssize_t udev_event_subst_format(
                 r = sd_device_get_devpath(dev, &val);
                 if (r < 0)
                         return r;
-                l = strpcpy(&s, l, val);
+                strpcpy(&s, l, val);
```

Comment:
do they need to be (void) casted, since in other cases they are checked?

Reply:
It does not return an error code. In that case, we usually silently ignore the return value. IIRC, it is documented in our coding rules.